pkgs/by-name/br/brother-dcpl2550dw/package.nix

```diff
@@ -0,0 +1,104 @@
+{ lib
+, stdenv
+, fetchurl
+, dpkg
+, autoPatchelfHook
+, makeWrapper
+, perl
+, gnused
+, ghostscript
+, file
+, coreutils
+, gnugrep
+, which
+}:
+
+let
+  arches = [ "x86_64" "i686" "armv7l" ];
+
+  runtimeDeps = [
```

Comment:
We could just inline this

Reply:
Is there an example you can point me to? Thanks

Reply:
I'm confident that means to just use the values of these variables in those places where use current use the variables.  This means the `let .. in` block  is no longer necessary.

Reply:
So do I move the variable declaration into the derivation? (without the let ... in part)